packages/frontend-2/components/project/CardImportFileArea.vue

```diff
@@ -69,18 +59,12 @@ const props = defineProps<{
 
 const {
   maxSizeInBytes,
-  onFilesSelected: onFilesSelectedInternal,
+  onFilesSelected,
   accept,
   upload: fileUpload,
   isUploading
 } = useFileImport(toRefs(props))
 
-const { showRegionStaticDataDisclaimer, triggerAction, onConfirmHandler } =
-  useWorkspaceCustomDataResidencyDisclaimerQuery({
```

Comment:
Do we need to keep these composables or can we remove them as well?

Reply:
@Mikehrn There's still one in use, and the other one is essentially a wrapper that may still be useful in the future